frontend/src/login/SignUpHandler.ts

```diff
@@ -0,0 +1,18 @@
+interface SignUpCredentials {
+    username: string;
+    password: string;
+}
+
+const SignUpHandler = ({ username, password }: SignUpCredentials) => {
+    console.log(`Signing up with username: ${username} and password: ${password}`);
+
+    if (!username || !password) {
+        alert('Username and password are required');
+        return;
+    }
+
+    alert(`User ${username} signed up successfully`);
+    localStorage.setItem('user', JSON.stringify({ username, password }));
```

Comment:
Username skal ikke sættes efter signup

Reply:
den bliver settet for at simulerer at den bliver gemt i DB'en

Reply:
Okay, et sign up flow skal ikke logge en bruger ind, ligemeget hvad der sker i DB

Reply:
Det her må stadig gerne fjernes med local storage